src/app/shared/object-grid/search-result-grid-element/item-search-result/item/item-search-result-grid-element.component.ts

```diff
@@ -23,9 +24,14 @@ export class ItemSearchResultGridElementComponent extends SearchResultGridElemen
    * Route to the item's page
    */
   itemPageRoute: string;
+  /**
+   * Whether to show the access status badge or not
+   */
+  showAccessStatus: boolean;
```

Comment:
Instead of repeated retrieval of `showAccessStatus` from config, can be retrieved and checked in `AccessStatusBadgeComponent`